### ISSUE TITLE
Panel: toggle button missing aria-label

### ISSUE THREAD

**Issue by djmj** [primefaces/primefaces#14734]
### Describe the bug

The toggle button in the `p:panel` is missing an aria-label.

In current showcase you can see the aria-label is added after clicking once on the toggle button (maybe because of the attached ajax handler?)

### Reproducer

See in current showcase: 

https://showcase.primefaces.org/ui/panel/panel.xhtml

### PrimeFaces edition

None

### PrimeFaces version

15.0.14

### JSF implementation

Mojarra

### JSF version

4.0.12

### Java version

17
